src/Aurora.Scrapers/Behaviours/FootfetishBooruBehaviour.cs

```diff
@@ -42,7 +42,7 @@ public static List<SearchItem> FootFetishBooruScrap(HtmlDocument document)
     private static FootfetishBooruResultData ExtractDataFrom(string termsStr)
     {
         var titleTerms = termsStr.Split(" ").Select(x => x.Trim());
-        List<string> tags = new List<string>();
+        List<string> tags = new();
```

Comment:
We already have FootfetishBooruResultData to wrap those values. We lose most of the typing data when using it, because of json codec, so it is not worth working with this